packet-solutions/xdp_prog_user.c

```diff
@@ -172,10 +172,6 @@ int main(int argc, char **argv)
 			fprintf(stderr, "can't write iface params\n");
 			return 1;
 		}
-	} else {
-		/* setup 1-1 mapping for the dynamic router */
-		for (i = 1; i < 256; ++i)
-			bpf_map_update_elem(map_fd, &i, &i, 0);
```

Comment:
Hmm, you should probably also update the kernel code to call `bpf_redirect()` instead of `bpf_redirect_map()` :)